### ISSUE TITLE
No warnings are shown when using class="blah" or classname="blah"

### ISSUE THREAD

**Issue by c0debreaker** [yannickcr/eslint-plugin-react#1198]
Here is my .eslintrc
```
{
  "extends": "react-app",
   "plugins": [
    "react"
  ],
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "rules": {
    "react/jsx-uses-react": "error",
    "react/jsx-uses-vars": "error",
  }
}
```

and I'm using Atom editor. I was hoping that the codes below would trigger an eslint warnings
```
<div class="test">test</div>
<div classname="test">test</div>
```
Once I close the div tag, the eslint warning at the bottom of the editor goes away

**Comment by ljharb** [comment]
React itself will issue those warnings.

**Comment by c0debreaker** [comment]
I see the issue in console in Chrome Dev Tools. However, I was hoping to see it during coding time. Eslint should take care of that right?

**Comment by ljharb** [comment]
No, eslint doesn't duplicate *every* runtime warning. Your tests have to handle that.

**Comment by c0debreaker** [comment]
Got it. Check out what I did. I commented out the usage of logo and look what eslint did. I was hoping I would also see something similar to this when I hover the mouse on `class="blah"`

http://imgur.com/a/UgoW8

**Comment by ljharb** [comment]
That's something static, defined in the file.

To warn on what you want, eslint-plugin-react would have to duplicate the list of valid DOM element props from inside React.

**Comment by yannickcr** [comment]
@c0debreaker I'm not sure what warning you expect ESLint to trigger here, but we have the [`no-unknown-property`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md) rule that can warn you on differences between HTML and JSX (like `className` instead of `class`)

**Comment by ljharb** [comment]
Oops, good call :-)

**Comment by c0debreaker** [comment]
Thank you folks!

**Comment by c0debreaker** [comment]
@yannickcr it worked really well! It's what I really want to happen :)

http://imgur.com/a/eTWZY
